fiftyone/utils/cvat.py

```diff
@@ -4372,152 +4378,163 @@ def download_annotations(self, results):
         annotations = {}
         deleted_tasks = []
 
-        for task_id in task_ids:
-            if not self.task_exists(task_id):
-                deleted_tasks.append(task_id)
-                logger.warning(
-                    "Skipping task %d, which no longer exists", task_id
-                )
-                continue
-
-            # Download task data
-            attr_id_map, _class_map_rev = self._get_attr_class_maps(task_id)
-            task_resp = self.get(self.task_annotation_url(task_id)).json()
-            all_shapes = task_resp["shapes"]
-            all_tags = task_resp["tags"]
-            all_tracks = task_resp["tracks"]
-
-            data_resp = self.get(self.task_data_meta_url(task_id)).json()
-            frames = data_resp["frames"]
-
-            label_fields = labels_task_map_rev[task_id]
-            label_types = self._get_return_label_types(
-                label_schema, label_fields
-            )
+        with fou.ProgressBar(
+            total=len(task_ids),
+            iters_str="tasks",
+            quiet=len(task_ids) == 1,
```

Comment:
nit: if `quiet` is not explicitly included as a kwarg to `ProgressBar`, then the default will be set based on `fo.config.show_progress_bars`. So, I would recommend passing `quiet=True` only when there's one task and, otherwise, not including `quiet` so that the `fo.config.show_progress_bars` will determine whether progress is shown